src/NServiceBus.Core.Tests/Routing/MessageDrivenSubscriptions/MessageDrivenSubscriptionsConfigExtensionsTests.cs

```diff
@@ -14,12 +15,13 @@
     using Transport;
 
     [TestFixture]
-    public class MessageDrivenSubscriptionsConfigExtensionsTests
+    public class RoutingSettingsTests
     {
         [Test]
         public void WhenPassingTransportAddressForPublisherInsteadOfEndpointName_ShouldThrowException()
         {
-            var routingSettings = new RoutingSettings<MessageDrivenTransportDefinition>(new SettingsHolder());
+            var settings = new SettingsHolder();
```

Comment:
do we need the settings instances as dedicated fields? Looks like you could just inline them again.